src/msg/async/AsyncConnection.cc

```diff
@@ -327,6 +327,8 @@ void AsyncConnection::process()
   bool already_dispatch_writer = false;
   std::lock_guard<std::mutex> l(lock);
   last_active = ceph::coarse_mono_clock::now();
+
+  rcu_quiescent_state();
```

Comment:
@yuyuyu101 Would this be a good place to call rcu_quiescent_state?  I think we need to do something when the thread is about to block (waiting on new connections?) to ensure it doesn't block the rcu write side, where would that be?